assets/layouts/default.vue

```diff
@@ -74,6 +85,9 @@ function showFuzzySearch() {
     active: true,
   });
 }
+function collapse() {
```

Comment:
I am not seeing why this moved to it's own function.

Reply:
There was a compilation error, so I took it out to the function.